packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -61,9 +62,26 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     private _pointerTravelDistance = 0;
     private _processSelectionOnUp = false;
     private _visibleRegionContainer: Container;
+    private _currLeft: number = 0;
+    private _currTop: number = 0;
     public get visibleRegionContainer() {
         return this._visibleRegionContainer;
     }
+    public get controlOffset() {

```

Comment:
You should remove this one :-) and instead export the const CONTROL_OFFSET.

On the other file, you could simply import the constant

Reply:
Oh that makes sense! But after talking to Patrick I think I can just completely remove the code from the other file :)